packages/node_modules/samples/browser-plugin-meetings/PROPOSAL.md

```diff
@@ -0,0 +1,77 @@
+# Meetings Plugin Kitchen Sink Reorganization Proposal
+
+## General Styling and Helper Queues
+
+- Each Logic Group Should be Collapsable / Disableable
+- The Presentational Order Should Match Group Order in this Document
+- Proper Presentation on Provided Listable Values Should be Implemented
+
+## Logic Grouping
+
+### Authentication
+
+- Allow User Token Input
+- Enables All Other Logic Groups Upon Successful Authentication
+
+### Meetings Management
+
+- Create Meetings
+- List All Meetings
+  - Join Option In List
+    - With Media
+    - Without Media
+  - Leave Option In List
```

Comment:
Maybe add a value to show the meeting that is currently joined?

Reply:
Added this.